vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -296,6 +304,44 @@ testFips() {
   echo "$test:Finish"
 }
 
+testCloudInit() {
+  test="testCloudInit"
+  echo "$test:Start"
+  os_sku=$1
+
+  # Limit this test only to Mariner or Azurelinux 
+  if [[ "$os_sku" == "CBLMariner" || "$os_sku" == "AzureLinux" ]]; then
```

Comment:
nit: I'm a stickler for surrounding variable names in braces. For example: `"${os_sku}"`. It's generally 'safer", because of potential weird naming and such.

That said, it looks to me like nothing here will blow up if you don't do that, so while I do recommend this, it isn't a big deal if you don't.